editor/ScriptLevelInterface.cpp

```diff
@@ -416,7 +416,7 @@ void CScriptLevelInterface::OnEditscript()
 			strcpy(buffer,"The specified path was not found.");
 			break;
 		case ERROR_BAD_FORMAT:  
-			strcpy(buffer,"The .exe file is invalid (non-Win32� .exe or error in .exe image).");
+			strcpy(buffer,"The .exe file is invalid (non-Win32� .exe or error in .exe image).");
```

Comment:
This still doesn't look right, at least here on GitHub. UTF8 issue?

Reply:
Yep, the same was done on other source files in the main code